django/db/backends/base/introspection.py

```diff
@@ -36,6 +36,26 @@ def identifier_converter(self, name):
         """
         return name
 
+    def only_nonempty_tables(self, tables):
```

Comment:
Unless merging into `.django_table_names()` with a `skip_empty` argument:
```suggestion
    def exclude_empty_tables(self, tables):
```